tests/schemas/cy/test_language.json

```diff
@@ -30,6 +30,7 @@
         {
```

Comment:
I guess this wasn't translated using the translation process as it is missing keys page title keys.

Reply:
Addressed in e3b8c52 for `cy` and `ga` schemas